src/common/actions/create-snap.js

```diff
@@ -54,10 +53,11 @@ function getSnapName(owner, name) {
     }));
 }
 
-export function createSnap(repositoryUrl, location) { // location for tests
+export function createSnap(repository) { // location for tests
```

Comment:
The "location for tests" comment is now misplaced - it should be moved to `createSnaps`, since that's where the `location` parameter now lives.